roc/receiver.go

```diff
@@ -30,6 +35,63 @@ func OpenReceiver(rocContext *Context, receiverConfig *ReceiverConfig) (*Receive
 	return (*Receiver)(receiver), nil
 }
 
+func (s *Receiver) Bind(portType PortType, proto Protocol, a *Address) error {
+	errCode := C.roc_receiver_bind(
+		(*C.roc_receiver)(s),
+		(C.roc_port_type)(portType),
+		(C.roc_protocol)(proto),
+		a.raw)
+	if errCode == 0 {
+		return nil
+	}
+	if errCode < 0 {
+		return ErrInvalidArguments // roc_receiver_bind returns negative in 3 cases: invalid args, cannot bind address and not enough resources, maybe we should consider renaming this error to a more generic name?
+	}
+	return ErrInvalidApi
+}
+
+func (r *Receiver) ReadAudioBytes(bytes []byte, samplesSize uint) error {
```

Comment:
I think right now reading or writing bytes doesn't make much sense. The only frame encoding supported so far is FrameEncodingPcmFloat, i.e. floats. So I suggest to remove bytes versions of read and write for now.